code/control/DeployPlanDispatcher.php

```diff
@@ -74,6 +74,7 @@ public function index(\SS_HTTPRequest $request) {
 	public function gitupdate(SS_HTTPRequest $request) {
 		switch($request->httpMethod()) {
 			case 'POST':
+				$this->checkSecurityToken(sprintf('%sSecurityID', get_class($this)));
```

Comment:
I'm not super happy with the result of a failing check is a text response, I was thinking of making changes to it so it would return a response that is more suitable for js consumption to display to the user.